src/mumble/MumbleApplication.cpp

```diff
@@ -25,9 +73,9 @@ MumbleApplication::MumbleApplication(int &pargc, char **pargv)
 }
 
 QString MumbleApplication::applicationVersionRootPath() {
-	QByteArray versionRoot = qgetenv("MUMBLE_VERSION_ROOT");
```

Comment:
So whose bug is this? This function not working like before surely breaks lots of other stuff out there.